Unable to build graylog anymore due to yarn build error

Hi,

I used to be able to build graylog locally without many troubles until now. The `mvn compile` always fails with this error:
```
[INFO]     ERROR in ./node_modules/graylog-web-plugin/plugin.js
[INFO]     Module not found: Error: Can't resolve './lib/PluginManifest' in '/Users/radykal/Projects/graylog-project-repos/graylog2-server/graylog2-web-interface/node_modules/graylog-web-plugin'
[INFO]      @ ./node_modules/graylog-web-plugin/plugin.js 2:18-49
[INFO]      @ ./src/index.jsx
[INFO]
[INFO]     ERROR in ./node_modules/graylog-web-plugin/plugin.js
[INFO]     Module not found: Error: Can't resolve './lib/PluginStore' in '/Users/radykal/Projects/graylog-project-repos/graylog2-server/graylog2-web-interface/node_modules/graylog-web-plugin'
[INFO]      @ ./node_modules/graylog-web-plugin/plugin.js 3:15-43
[INFO]      @ ./src/index.jsx
[INFO]     Child html-webpack-plugin for "index.html":
[INFO]          1 asset
[INFO]         Entrypoint undefined = index.html
[INFO]         [9YbPW59L] (webpack)/buildin/module.js 497 bytes {639f79af-0} [built]
[INFO]         [aY11sVfy] (webpack)/buildin/global.js 472 bytes {639f79af-0} [built]
[INFO]         [i/dVZhsi] ./node_modules/html-webpack-plugin/lib/loader.js!./templates/index.html.template 1.52 KiB {639f79af-0} [built]
[INFO]             + 1 hidden module
[INFO]     Child html-webpack-plugin for "module.json":
[INFO]          1 asset
[INFO]         Entrypoint undefined = module.json
[INFO]         [1S/p2sYH] ./node_modules/html-webpack-plugin/lib/loader.js!./templates/module.json.template 379 bytes {639f79af-0} [built]
[INFO]         [9YbPW59L] (webpack)/buildin/module.js 497 bytes {639f79af-0} [built]
[INFO]         [aY11sVfy] (webpack)/buildin/global.js 472 bytes {639f79af-0} [built]
[INFO]             + 1 hidden module
[INFO] error Command failed with exit code 2.
[INFO] error Command failed with exit code 2.info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for graylog-project 3.3.0-SNAPSHOT:
[INFO]
[INFO] Graylog Parent POM ................................. SUCCESS [  0.003 s]
[INFO] Graylog Project Parent POM ......................... SUCCESS [  0.123 s]
[INFO] Graylog ............................................ FAILURE [02:56 min]
```

My local env is as follows:
```
[radykal@macbook-pro.local]➜ ~/Projects/graylog-project(master|…) $ graylog-project version
Version 0.22.0, built on 2019-07-20T08:54:07+0000 (revision 1ef2728)
```

```
[radykal@macbook-pro.local]➜ ~/Projects/graylog-project(master|…) $ graylog-project status
Current project status
  Manifests: [manifests/master.json]
  Module versions
    graylog-parent               3.3.0-SNAPSHOT (branch: master)
    graylog-plugin-collector     3.3.0-SNAPSHOT (branch: master, parent: 3.3.0-SNAPSHOT)
    graylog-plugin-aws           3.3.0-SNAPSHOT (branch: master, parent: 3.3.0-SNAPSHOT)
    graylog-plugin-threatintel   3.3.0-SNAPSHOT (branch: master, parent: 3.3.0-SNAPSHOT)
    graylog-plugin-integrations  3.3.0-SNAPSHOT (branch: master, parent: 3.3.0-SNAPSHOT)
```

The only issue I found with similar error is this one https://community.graylog.org/t/request-help-for-the-contribution-documentation/898/10
So I ran `graylog-project npm-clean` to delete all node and node_modules dirs from all repos but same error appears.

Dunno what to do, I was about to help with some open issues but after 2+ hours trying to fix the build to even start is very frustrating. Can someone help?

--- a/graylog2-web-interface/packages/graylog-web-plugin/package.json
+++ b/graylog2-web-interface/packages/graylog-web-plugin/package.json
@@ -34,7 +34,7 @@
     "README.md"
   ],
   "dependencies": {
-    "@babel/preset-env": "7.8.4",
+    "@babel/preset-env": "7.8.7",
     "babel-eslint": "9.0.0",
     "eslint-config-graylog": "file:../eslint-config-graylog",
     "html-webpack-plugin": "3.2.0",
@@ -59,7 +59,7 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.8.0",
-    "@babel/core": "^7.8.0",
+    "@babel/core": "^7.8.7",
     "babel-plugin-add-module-exports": "^1.0.2"
   }
 }

